src/Model/Filter/Like.php

```diff
@@ -143,26 +155,34 @@ protected function _wildcards($value)
      */
     protected function _formatWildcards($value)
     {
-        $from = $to = $substFrom = $substTo = [];
-        if ($this->config('wildcardAny') !== '%') {
-            $from[] = '%';
-            $to[] = '\%';
-            $substFrom[] = $this->config('wildcardAny');
-            $substTo[] = '%';
-        }
-        if ($this->config('wildcardOne') !== '_') {
-            $from[] = '_';
-            $to[] = '\_';
-            $substFrom[] = $this->config('wildcardOne');
-            $substTo[] = '_';
+        $value = $this->_escapeDriver->formatWildcards($value);
+
+        return $value;
+    }
+
+    /**
+     * set configuration for escape driver name
+     *
+     * @return void
+     */
+    protected function _setEscapeDriver()
+    {
+        if ($this->config('escapeDriver') === null) {
```

Comment:
Should this be "escaper" for consistency with the class name? Do we need the word "driver" in here? //cc @ADmad